install.sh

```diff
@@ -0,0 +1,211 @@
+#!/bin/bash
+
+set -u
+
+DOTENV_LINTER="dotenv-linter"
+DOTENV_LINTER_REPO="${DOTENV_LINTER}/${DOTENV_LINTER}"
+DOTENV_LINTER_GITHUB="https://github.com/${DOTENV_LINTER_REPO}"
+DOTENV_LINTER_RELEASE_API="https://api.github.com/repos/${DOTENV_LINTER_REPO}/releases/latest"
+
+main() {
+    require_cmd uname
+    require_cmd mkdir
+    require_cmd grep
+    require_cmd cut
+    require_cmd rm
+
+    local _target_version
+    if [ $# -eq 0 ]; then
```

Comment:
Can you tell me more about that?

Reply:
What exactly are you interested in?
`$#` - the number of arguments, if a specific version is not specified, we get the last by means of API github.

Without the need for a multi-version installation, I will also simplify downloading the latest version (remove the extra github api call)